### PR TITLE
OCM-8988 | test:Automated id:57105,59430,67412

### PR DESCRIPTION
**Pull request by xueli181114**
```
lixue@Xue-Lis-MacBook-Pro rosa % ginkgo -focus "(67412|57105|59430)" tests/e2e
Running Suite: ROSA CLI e2e tests suite - /Users/lixue/Workspace/rosa/tests/e2e
===============================================================================
Random Seed: 1721386246

Will run 3 of 149 specs
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 1 of 149 Specs in 52.093 seconds
SUCCESS! -- 1 Passed | 0 Failed | 0 Pending | 148 Skipped
PASS

Ginkgo ran 1 suite in 59.091257981s
Test Suite Passed
lixue@Xue-Lis-MacBook-Pro rosa % echo $TEST_PROFILE
rosa-advanced
```

### CODE REVIEW

**Comment by xueli181114** [comment]
```
lixue@Xue-Lis-MacBook-Pro rosa % ginkgo -focus "(67412|57105|59430)" tests/e2e
Running Suite: ROSA CLI e2e tests suite - /Users/lixue/Workspace/rosa/tests/e2e
===============================================================================
Random Seed: 1721386132

Will run 3 of 149 specs
SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS•SSSSS•SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS

Ran 2 of 149 Specs in 899.851 seconds
SUCCESS! -- 2 Passed | 0 Failed | 0 Pending | 147 Skipped
PASS

Ginkgo ran 1 suite in 15m15.522786813s
Test Suite Passed
lixue@Xue-Lis-MacBook-Pro rosa % echo $TEST_PROFILE 
rosa-hcp-upgrade-y-stream
lixue@Xue-Lis-MacBook-Pro rosa % rosa version
I: 1.2.42
I: Your ROSA CLI is up to date.
```

File: tests/e2e/test_rosacli_machine_pool.go [hunk 1]
```diff
@@ -1076,4 +1076,70 @@ var _ = Describe("Edit machinepool",
 				Expect(mpDescription.AutoScaling).To(Equal(constants.Yes))
 				Expect(mpDescription.Replicas).To(Equal("3-12"))
 			})
+
+		It("will validate lables and taints for default worker pool - [id:57105]", labels.Runtime.Day2, labels.Medium,
```
Comment (radtriste) [inline tests/e2e/test_rosacli_machine_pool.go:1080]:
```suggestion
		It("will validate labels and taints for default worker pool - [id:57105]", labels.Runtime.Day2, labels.Medium,
```

**Review by radtriste — APPROVED** [review]
small fix

**Pull request merged**